app/Domain/Goalcanvas/Templates/showCanvas.blade.php

```diff
@@ -145,32 +145,44 @@ class="delete">
                     <div class="pull-right">
                         <div class="btn-group viewDropDown">
                             @if (count($allCanvas) > 0 && !empty($statusLabels))
-                                @if ($filter['status'] == 'all')
-                                    <button class="btn dropdown-toggle" data-toggle="dropdown">
-                                        <i class="fas fa-filter"></i> {!! __('status.all') !!} {!! __('links.view') !!}
-                                    </button>
-                                @else
-                                    <button class="btn dropdown-toggle" data-toggle="dropdown">
-                                        <i class="fas fa-fw {{ __($statusLabels[$filter['status']]['icon']) }}"></i>
-                                        {{ $statusLabels[$filter['status']]['title'] }} {{ __('links.view') }}
-                                    </button>
-                                @endif
-                                <ul class="dropdown-menu">
-                                    <li>
-                                        <a href="{{ BASE_URL }}/goalcanvas/showCanvas?filter_status=all"
-                                            @if ($filter['status'] == 'all') class="active" @endif>
+                                @php
+                                    if ($filter['status'] == 'all') {
+                                        $labelText =
+                                            '<i class="fas fa-filter"></i> ' .
+                                            __('status.all') .
+                                            ' ' .
+                                            __('links.view');
+                                    } else {
+                                        $labelText =
+                                            '<i class="fas fa-fw ' .
+                                            __($statusLabels[$filter['status']]['icon']) .
+                                            '"></i> ' .
+                                            $statusLabels[$filter['status']]['title'] .
+                                            ' ' .
+                                            __('links.view');
+                                    }
+                                @endphp
+
+                                <x-global::actions.dropdown :labelText="html_entity_decode($labelText)" class="btn dropdown-toggle" align="start"
```

Comment:
Can you update this to use <x-slot:label-Text> and remove the html_entity_decode?